ethereum/p2p/src/test/java/tech/pegasys/pantheon/ethereum/p2p/network/DefaultP2PNetworkTest.java

```diff
@@ -201,31 +184,6 @@ public void checkMaintainedConnectionPeers_connectingPeer() {
     verify(rlpxAgent, times(0)).connect(peer);
   }
 
-  @Test
-  public void whenBuildingNetworkWithNodePermissioningWithoutBlockchainShouldThrowIllegalState() {
```

Comment:
It is no longer possible to construct the corresponding `PeerPermissions` object without supplying the nodePermissioning and blockchain objects.